<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="68065" started-at="2018-09-17T11:47:05Z" finished-at="2018-09-17T11:48:13Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="68065" started-at="2018-09-17T11:47:05Z" finished-at="2018-09-17T11:48:13Z">
      <class name="com.qa.Homepage.CreateContact">
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.qa.Homepage.CreateContact@5f341870]" name="Setup" is-config="true" duration-ms="7699" started-at="2018-09-17T13:47:43Z" finished-at="2018-09-17T13:47:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="CreateContactTest()[pri:0, instance:com.qa.Homepage.CreateContact@5f341870]" name="CreateContactTest" duration-ms="22243" started-at="2018-09-17T13:47:51Z" finished-at="2018-09-17T13:48:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateContactTest -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.qa.Homepage.CreateContact@5f341870]" name="TearDown" is-config="true" duration-ms="91" started-at="2018-09-17T13:48:13Z" finished-at="2018-09-17T13:48:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.qa.Homepage.CreateContact -->
      <class name="com.qa.Homepage.FailedTestCase">
        <test-method status="FAIL" signature="test()[pri:0, instance:com.qa.Homepage.FailedTestCase@553f17c]" name="test" duration-ms="1" started-at="2018-09-17T13:48:13Z" finished-at="2018-09-17T13:48:13Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at com.qa.Homepage.FailedTestCase.test(FailedTestCase.java:11)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.qa.Homepage.FailedTestCase -->
      <class name="com.qa.Homepage.LoginTest">
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.qa.Homepage.LoginTest@14bf9759]" name="Setup" is-config="true" duration-ms="8546" started-at="2018-09-17T13:47:05Z" finished-at="2018-09-17T13:47:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="ContactTest()[pri:0, instance:com.qa.Homepage.LoginTest@14bf9759]" name="ContactTest" duration-ms="7628" started-at="2018-09-17T13:47:14Z" finished-at="2018-09-17T13:47:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ContactTest -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.qa.Homepage.LoginTest@14bf9759]" name="TearDown" is-config="true" duration-ms="2090" started-at="2018-09-17T13:47:22Z" finished-at="2018-09-17T13:47:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.qa.Homepage.LoginTest@14bf9759]" name="Setup" is-config="true" duration-ms="12439" started-at="2018-09-17T13:47:24Z" finished-at="2018-09-17T13:47:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="VerifyLogin()[pri:0, instance:com.qa.Homepage.LoginTest@14bf9759]" name="VerifyLogin" duration-ms="7209" started-at="2018-09-17T13:47:36Z" finished-at="2018-09-17T13:47:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLogin -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.qa.Homepage.LoginTest@14bf9759]" name="TearDown" is-config="true" duration-ms="72" started-at="2018-09-17T13:47:43Z" finished-at="2018-09-17T13:47:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.qa.Homepage.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
